tests/operator_test.go

```diff
@@ -1019,8 +1019,7 @@ spec:
 	BeforeEach(func() {
 		tests.BeforeTestCleanup()
 
-		workDir, err = ioutil.TempDir("", tests.TempDirPrefix+"-")
-		Expect(err).ToNot(HaveOccurred())
+		workDir = tests.TempDir()
```

Comment:
It might be overkill to declare GinkgoT for that but please consider using GinkgoT().TempDir()
It is auto cleaned, and it is context aware of the BeforeEach if u set it there (or in the subject node itself, i.e the `It`)

Alternatively to use this something like
```
	tmpDir, err := ioutil.TempDir("","pattern")
	Expect(err).ToNot(HaveOccurred())
	DeferCleanup(os.RemoveAll, tmpDir)
```
and then you can remove AfterEach, 
since we are now using Ginkgo 2.0, it is supported

Reply:
Oh, I did not notice that Ginkgo 2.0 is in, congratulations!
@oshoval, I prefer doing as you suggest, but why do you consider `GinkgoT` an "overkill"?

Reply:
Just because we don't actually use it's control to mock thing, and only declaring it, with all its helpers for a tempDir,
but fine with me if you prefer it, thanks